grunt/tasks/create-json-config.js

```diff
@@ -34,6 +34,6 @@ module.exports = function(grunt) {
             }
         });
 
-        grunt.file.write(grunt.config('outputdir') + 'course/config.json', JSON.stringify(configJson));
+        grunt.file.write(grunt.config('outputdir') + 'course/config.json', JSON.stringify(configJson, null, "    "));
```

Comment:
Would you mind using the numerical argument instead i.e. `4`? More readable IMO.